core-blocks/shortcode/index.js

```diff
@@ -46,7 +47,7 @@ export const settings = {
 					text: {
 						type: 'string',
 						shortcode: ( attrs, { content } ) => {
-							return content;
+							return removep( autop( content ) );
```

Comment:
It's not evident why we're doing this. An inline code comment would have been appropriate.